docs/src/components/landing/Hero.js

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import IconImage from 'docs/src/components/icon/IconImage';
+import GradientText from './GradientText';
+import SignUp from './SignUp';
+import ToolpadHeroContainer from '../../layouts/ToolpadHeroContainer';
+import Demo from './Demo';
+
+export default function Hero() {
+  return (
+    <ToolpadHeroContainer>
+      <Box sx={{ minHeight: { xs: '0px', sm: '20px' } }} />
```

Comment:
We should probably use the padding or the margin instead of creating a new DOM node

```suggestion
```